### ISSUE TITLE
Dinky script support

### ISSUE THREAD

**Issue by markusbkk** [scemino/engge2#3]
Curious. Are there any plans to support Dinky at some point in the future?

Thanks to Ron Gilbert's [Delores](https://github.com/grumpygamer/DeloresDev) there's some documentation and example code.

**Comment by scemino** [comment]
> Curious. Are there any plans to support Dinky at some point in the future?

No, this is not planned, except if someone wants to help me on this task and if it's worth it.

> Thanks to Ron Gilbert's [Delores](https://github.com/grumpygamer/DeloresDev) there's some documentation and example code.

Yes... but no, the main problem is the .dink file has not been reversed yet completely.

**Comment by markusbkk** [comment]
I definitely feel like it's worth it. Not necessarily to play back the Delores demo or even get Return to Monkey Island working, but as a nice alternative to existing adventure engines for game designers to tinker with.

I'd definitely be willing to assist in the planning and implementation, too. Time permitting (RN I work towards modernizing ScummC).

**Comment by scemino** [comment]
I'm not sure about it, I made `engge` and I explained how to create your own adventure game with it: https://scemino.github.io/create-room/ nobody seems to be interested.

**Comment by markusbkk** [comment]
> I'm not sure about it, I made `engge` and I explained how to create your own adventure game with it: https://scemino.github.io/create-room/ nobody seems to be interested.

I saw that post. It's actually what partly inspired this GitHub Issue.
I absolutely love the idea. I think the main problem with the current method is that it essentially requires Thimbleweed Park as a "donor game".
